docs/products/opensearch/dashboards/index.rst

```diff
@@ -0,0 +1,31 @@
+OpenSearch Dashboards
+======================
+
+OpenSearch Dashboards is both a visualisation tool for data inside an OpenSearch cluster and a user interface for OpenSearch plugins.
+
+.. note::
+    OpenSearch Dashboards was forked in 2021 from the formerly open source project Kibana.
+
+Get started with Aiven for OpenSearch Dashboards
+------------------------------------------------
+
+Take your first steps with Aiven for OpenSearch Dashboards by following our :doc:`getting-started` article, or browse through the full list of articles:
+
+.. panels::
+
+    💻 :doc:`howto`
+
+
+Ways to use OpenSearch Dashboards
+---------------------------------
+
+OpenSearch Dashboards is ideal for data visualisation and monitoring. It provides a possibility to set up dashboards using visual tools, an interface to directly send DSL queries and run SQL queries.
```

Comment:
A bit of the wording: "It provides a possibility to set up dashboards using visual tools..." -> "It enables building dashboards using visual tools providing an interface to send DSL and SQL queries" 

Maybe it's me not knowing the details but why is "sending" DSL and "running" SQL? can't both be run?

Reply:
You're right! They mean the same :) I've changed this!